### PR TITLE
JENKINS-43256# SSE subscribe event includes organization

### PR DESCRIPTION
**Pull request by vivek**
# Description

See [JENKINS-43256](https://issues.jenkins-ci.org/browse/JENKINS-43256).

To test check the network tab, sse subscribe event should include jenkins_org element:
```
{
   "subscribe" : [
      {
         "jenkins_org" : "jenkins",
         "jenkins_channel" : "job"
      },
      {
         "jenkins_channel" : "job",
         "jenkins_org" : "jenkins"
      }
   ],
   "unsubscribe" : [],
   "dispatcherId" : "jenkins-blueocean-core-js-1492722251735-3cxgmk"
}
```
@tfennelly @cliffmeyers PTAL. For some reason there are duplicate job channel subscription but thats how it's always been and beyond scope of this PR.

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-core-js/src/js/sse/SseBus.js [hunk 1]
```diff
@@ -44,7 +45,7 @@ export class SseBus {
         if (!this.sseListeners.job) {
             const sseListener = this.connection.subscribe('job', (event) => {
                 this._handleJobEvent(event);
-            });
+            }, { jenkins_org: AppConfig.getOrganizationName() });
```
Comment (michaelneale) [inline blueocean-core-js/src/js/sse/SseBus.js]: so how does this work - is this filtering events coming back or throwing away events not for the current org - if the latter, this wont' really be a scalable approach if there are lot of orgs right?

Comment (vivek) [inline blueocean-core-js/src/js/sse/SseBus.js]: Each UI instance knows one org. All we are telling is backend SSE pub-sub to only return events for this org. So in multi-master case, there can be special pub sub implementation that would filter events accordingly. Why you think scalability could be an issue?

**Comment by michaelneale** [comment]
This will need to be updated to master please

File: blueocean-core-js/src/js/sse/SseBus.js (hunk 1 above)
Comment (michaelneale) [inline blueocean-core-js/src/js/sse/SseBus.js]: ok cool

**Comment by michaelneale** [comment]
Can this be updated to master? also - if the branch name can have a little slug of what the ticket is about, helps with reviewing.

Does this pass ATH? I think that and @tfennelly should be enough to allow this in.

**Comment by vivek** [comment]
Its late for me, tomorrow will update it to master.

>also - if the branch name can have a little slug of what the ticket is about, helps with reviewing.

Heh,  doesn't PR has link to associated ticket and PR description is already there, no?

**Comment by vivek** [comment]
Merged latest master. Lets see CI, ATH.

**Comment by michaelneale** [comment]
@vivek oh I just have a lot of things checked out - and see messages about runs, but only has the ticket number so I don't remember what one is what when looking at a listing.

**Comment by vivek** [comment]
@michaelneale ah I see, right checked out branches its hard. doesn't hurt to include some description.

**Comment by i386** [comment]
Please wait for #979 to land in master.

**Comment by michaelneale** [comment]
if I get a better memory for numbers I wouldn't have this problem... there is also some shell script foo that makes it show up (but also in hipchat/other messages too)

**Comment by michaelneale** [comment]
Looks ok - updaste to master and check ATH is ok and :bee:

**Comment by cliffmeyers** [comment]
@vivek LGTM to me too. I believe the reason there are two subscriptions is that personalization makes it own subscription to SSE as at the time there was no mechanism to share a model or data between the two. Once we get ATH for personalization we can refactor to share state from dashboard / core-js and can eliminate this subscription.

**Comment by vivek** [comment]
@cliffmeyers sounds good. 

@i386 #979 looks like ready to land?

**Comment by michaelneale** [comment]
once updated and builds happy etc - sure. Go for it.

**Comment by vivek** [comment]
ATH passes https://ci.blueocean.io/blue/organizations/jenkins/ATH-Jenkinsfile/detail/master/2303/pipeline.

**Pull request merged**